templates/alltube/meta.yaml

```diff
@@ -0,0 +1,48 @@
+name: AllTube
+description: Web GUI for youtube-dl
+instructions: null
+changeLog:
+  - date: 2023-07-11
+    description: First Release
+links:
+  - label: Github
+    url: https://github.com/Rudloff/alltube
+  - label: Docker
+    url: https://hub.docker.com/r/dnomd343/alltube
+contributors:
+  - name: Berk Sümbül
+    url: https://berksmbl.com
+schema:
+  type: object
+  required:
+    - projectName
+    - appServiceName
+    - appServiceImage
+  properties:
+    projectName:
+      type: string
+      title: Project Name
+    alltubeTitle:
+      type: string
+      title: Title
+      default: My AllTube Site
+    alltubeConvert:
+      type: boolean
+      title: Convert
+      default: true
+    alltubeStream:
+      type: boolean
+      title: Stream
+      default: true
+    alltubeRemux:
+      type: boolean
+      title: Remux
+      default: true
+    appServiceName:
+      type: string
+      title: App Service Name
+      default: alltube
+    appServiceImage:
+      type: string
+      title: App Service Image
+      default: dnomd343/alltube:3.2.0-alpha
```

Comment:
this tag is much older then latest, please revert

Reply:
I got it back. @deiucanta